prep-javascript-dom-events/prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+
+    document.addEventListener('DOMContentLoaded', function () {
+      console.log('content loaded!');
+    });
+
+    window.addEventListener('mousemove', function () {
+      console.log('mouse is moving!');
+    });
+
+    var $helloWorld = document.querySelector('button.hello-world-btn');
+    $helloWorld.addEventListener = ('click');
+    console.log('hello world!');
```

Comment:
Hi Ralla, you want do add an event listener similar to the `document` and `window`.

```js
$helloWorld.addEventListener('click', function () {

})
```